crates/icondiffbot2/src/main.rs

```diff
@@ -98,8 +98,6 @@ const JOB_JOURNAL_LOCATION: &str = "jobs";
 
 #[actix_web::main]
 async fn main() -> std::io::Result<()> {
-    std::env::set_current_dir(std::env::current_exe().unwrap().parent().unwrap()).unwrap();
-
     stable_eyre::install().expect("Eyre handler installation failed!");
```

Comment:
Yeah I dont think this is a good idea

The whole reason we did this check is because of the directory swapping with sdmm and other stuff so much, and it kept making repo clones inside other repos and all sorts of unpleasant business.

I can try running this but I seem to remember needing a ton of path sets for this to be safe.

Reply:
it doesn't set the cwd willy nilly anymore, so this should be fine

Reply:
> it doesn't set the cwd willy nilly anymore, so this should be fine

Do you have this much faith in sdmm?